### PR TITLE
Add mocha gem for test stubbing

### PR DESCRIPTION
**Pull request by veronicacannon**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -318,6 +318,7 @@ DEPENDENCIES
   license_finder (~> 5.0.0)
   listen (~> 3.1.5)
   mini_magick (~> 4.8.0)
+  mocha (~> 1.3.0)
```
Comment (BrendaPenn) [inline Gemfile.lock]: @veronicacannon Shouldn't the mocha gem be added to the Gemfile in addition to the Gemfile.lock?

Comment (elskwid) [reply Gemfile.lock]: @BrendaPenn yes! The workflow is: 1) add file to Gemfile, 2) run `bundle install`(this step modifies the Gemfile.lock)

Comment (elskwid) [reply Gemfile.lock]: (sorry for jumping in 😸 )

Comment (veronicacannon) [reply Gemfile.lock]: Sorry I missed this @BrendaPenn Thanks @elskwid

Comment (veronicacannon) [reply Gemfile.lock]: Rebased to latest Gemfile and ready for approval.

**Pull request merged**